tests/test_dataserializer.py

```diff
@@ -0,0 +1,38 @@
+"""
```

Comment:
I'd recommend renaming this module to `test_data_serializer.py` (with an underscore between `data` and `serializer`) because that's the name of the tested module.